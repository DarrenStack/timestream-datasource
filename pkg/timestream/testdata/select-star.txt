Frame[0] {
    "custom": {
        "queryId": "{CHANGES}"
    },
    "stats": []
}
Name: 
Dimensions: 15 Fields by 10 Rows
+-------------------------+-------------------------+--------------------------------------+--------------------+------------------+-------------------+------------------+-----------------+-------------------------+---------------------+-----------------------------+-----------------------------+------------------------------+--------------------+-------------------------------+
| Name: availability_zone | Name: microservice_name | Name: instance_name                  | Name: process_name | Name: os_version | Name: jdk_version | Name: cell       | Name: region    | Name: silo              | Name: instance_type | Name: measure_value::double | Name: measure_value::bigint | Name: measure_value::varchar | Name: measure_name | Name: time                    |
| Labels:                 | Labels:                 | Labels:                              | Labels:            | Labels:          | Labels:           | Labels:          | Labels:         | Labels:                 | Labels:             | Labels:                     | Labels:                     | Labels:                      | Labels:            | Labels:                       |
| Type: []*string         | Type: []*string         | Type: []*string                      | Type: []*string    | Type: []*string  | Type: []*string   | Type: []*string  | Type: []*string | Type: []*string         | Type: []*string     | Type: []*float64            | Type: []*int64              | Type: []*string              | Type: []*string    | Type: []time.Time             |
+-------------------------+-------------------------+--------------------------------------+--------------------+------------------+-------------------+------------------+-----------------+-------------------------+---------------------+-----------------------------+-----------------------------+------------------------------+--------------------+-------------------------------+
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 90.25268864587973           | null                        | null                         | memory_used        | 2020-06-05 15:15:51 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 19.903028203271756          | null                        | null                         | memory_used        | 2020-06-05 15:18:02 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 49.378959923123254          | null                        | null                         | memory_used        | 2020-06-05 15:19:57 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 32.972727452625115          | null                        | null                         | memory_used        | 2020-06-05 15:21:53 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 53.81475840832846           | null                        | null                         | memory_used        | 2020-06-05 15:23:49 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 35.02757589815768           | null                        | null                         | memory_used        | 2020-06-05 15:25:43 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 75.44833507353106           | null                        | null                         | memory_used        | 2020-06-05 15:27:37 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 45.05576699624557           | null                        | null                         | memory_used        | 2020-06-05 15:29:33 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 93.23989111380858           | null                        | null                         | memory_used        | 2020-06-05 15:31:30 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 15.600054779879324          | null                        | null                         | memory_used        | 2020-06-05 15:33:27 +0000 UTC |
+-------------------------+-------------------------+--------------------------------------+--------------------+------------------+-------------------+------------------+-----------------+-------------------------+---------------------+-----------------------------+-----------------------------+------------------------------+--------------------+-------------------------------+



