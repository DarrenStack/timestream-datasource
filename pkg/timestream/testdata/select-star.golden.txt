ðŸŒŸ This was machine generated.  Do not edit. ðŸŒŸ

Frame[0] {}
Name: 
Dimensions: 15 Fields by 10 Rows
+-------------------------+-------------------------+--------------------------------------+--------------------+------------------+-------------------+------------------+-----------------+-------------------------+---------------------+-----------------------------+-----------------------------+------------------------------+--------------------+-------------------------------+
| Name: availability_zone | Name: microservice_name | Name: instance_name                  | Name: process_name | Name: os_version | Name: jdk_version | Name: cell       | Name: region    | Name: silo              | Name: instance_type | Name: measure_value::double | Name: measure_value::bigint | Name: measure_value::varchar | Name: measure_name | Name: time                    |
| Labels:                 | Labels:                 | Labels:                              | Labels:            | Labels:          | Labels:           | Labels:          | Labels:         | Labels:                 | Labels:             | Labels:                     | Labels:                     | Labels:                      | Labels:            | Labels:                       |
| Type: []*string         | Type: []*string         | Type: []*string                      | Type: []*string    | Type: []*string  | Type: []*string   | Type: []*string  | Type: []*string | Type: []*string         | Type: []*string     | Type: []*float64            | Type: []*int64              | Type: []*string              | Type: []*string    | Type: []time.Time             |
+-------------------------+-------------------------+--------------------------------------+--------------------+------------------+-------------------+------------------+-----------------+-------------------------+---------------------+-----------------------------+-----------------------------+------------------------------+--------------------+-------------------------------+
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 93.80957301609632           | null                        | null                         | memory_used        | 2020-06-03 06:53:35 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 5.061653699799118           | null                        | null                         | memory_used        | 2020-06-03 06:56:10 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 26.04487919220061           | null                        | null                         | memory_used        | 2020-06-03 06:58:31 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 57.932903032943564          | null                        | null                         | memory_used        | 2020-06-03 07:00:53 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 81.97032210228853           | null                        | null                         | memory_used        | 2020-06-03 07:03:17 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 82.6186066172801            | null                        | null                         | memory_used        | 2020-06-03 07:05:39 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 18.100376469399304          | null                        | null                         | memory_used        | 2020-06-03 07:08:00 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 15.443329409581319          | null                        | null                         | memory_used        | 2020-06-03 07:10:32 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 12.51647055510372           | null                        | null                         | memory_used        | 2020-06-03 07:12:51 +0000 UTC |
| us-east-1-1             | apollo                  | i-zaZswmJk-apollo-0000.amazonaws.com | null               | AL2              | null              | us-east-1-cell-1 | us-east-1       | us-east-1-cell-1-silo-1 | r5.4xlarge          | 97.53213817182794           | null                        | null                         | memory_used        | 2020-06-03 07:15:13 +0000 UTC |
+-------------------------+-------------------------+--------------------------------------+--------------------+------------------+-------------------+------------------+-----------------+-------------------------+---------------------+-----------------------------+-----------------------------+------------------------------+--------------------+-------------------------------+


====== TEST DATA RESPONSE (arrow base64) ======
FRAME=QVJST1cxAAD/////QAcAABAAAAAAAAoADgAMAAsABAAKAAAAFAAAAAAAAAEDAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAZPn//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAACE+f//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAKT5//8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAADwAAABgGAACUBQAAKAUAALwEAABYBAAA9AMAAJgDAAA8AwAA4AIAAHQCAADsAQAAYAEAAOQAAAB4AAAAGAAAAAAAEgAYABQAAAATAAwAAAAIAAQAEgAAABQAAAA8AAAAPAAAAAAAAAo8AAAAAQAAAAQAAAA4+v//CAAAABAAAAAEAAAAdGltZQAAAAAEAAAAbmFtZQAAAAAAAAAAKv7//wAAAwAEAAAAdGltZQAAAACm+v//FAAAAEQAAABEAAAAAAAFAUAAAAABAAAABAAAAJT6//8IAAAAGAAAAAwAAABtZWFzdXJlX25hbWUAAAAABAAAAG5hbWUAAAAAAAAAAIj6//8MAAAAbWVhc3VyZV9uYW1lAAAAAA77//8UAAAATAAAAEwAAAAAAAUBSAAAAAEAAAAEAAAA/Pr//wgAAAAgAAAAFgAAAG1lYXN1cmVfdmFsdWU6OnZhcmNoYXIAAAQAAABuYW1lAAAAAAAAAAD4+v//FgAAAG1lYXN1cmVfdmFsdWU6OnZhcmNoYXIAAIb7//8UAAAATAAAAFQAAAAAAAIBWAAAAAEAAAAEAAAAdPv//wgAAAAgAAAAFQAAAG1lYXN1cmVfdmFsdWU6OmJpZ2ludAAAAAQAAABuYW1lAAAAAAAAAAAIAAwACAAHAAgAAAAAAAABQAAAABUAAABtZWFzdXJlX3ZhbHVlOjpiaWdpbnQAAAAO/P//FAAAAEwAAABUAAAAAAADAVQAAAABAAAABAAAAPz7//8IAAAAIAAAABUAAABtZWFzdXJlX3ZhbHVlOjpkb3VibGUAAAAEAAAAbmFtZQAAAAAAAAAAAAAGAAgABgAGAAAAAAACABUAAABtZWFzdXJlX3ZhbHVlOjpkb3VibGUAAACS/P//FAAAAEQAAABEAAAAAAAFAUAAAAABAAAABAAAAID8//8IAAAAGAAAAA0AAABpbnN0YW5jZV90eXBlAAAABAAAAG5hbWUAAAAAAAAAAHT8//8NAAAAaW5zdGFuY2VfdHlwZQAAAPr8//8UAAAAPAAAADwAAAAAAAUBOAAAAAEAAAAEAAAA6Pz//wgAAAAQAAAABAAAAHNpbG8AAAAABAAAAG5hbWUAAAAAAAAAANT8//8EAAAAc2lsbwAAAABS/f//FAAAADwAAAA8AAAAAAAFATgAAAABAAAABAAAAED9//8IAAAAEAAAAAYAAAByZWdpb24AAAQAAABuYW1lAAAAAAAAAAAs/f//BgAAAHJlZ2lvbgAAqv3//xQAAAA8AAAAPAAAAAAABQE4AAAAAQAAAAQAAACY/f//CAAAABAAAAAEAAAAY2VsbAAAAAAEAAAAbmFtZQAAAAAAAAAAhP3//wQAAABjZWxsAAAAAAL+//8UAAAAQAAAAEAAAAAAAAUBPAAAAAEAAAAEAAAA8P3//wgAAAAUAAAACwAAAGpka192ZXJzaW9uAAQAAABuYW1lAAAAAAAAAADg/f//CwAAAGpka192ZXJzaW9uAGL+//8UAAAAQAAAAEAAAAAAAAUBPAAAAAEAAAAEAAAAUP7//wgAAAAUAAAACgAAAG9zX3ZlcnNpb24AAAQAAABuYW1lAAAAAAAAAABA/v//CgAAAG9zX3ZlcnNpb24AAML+//8UAAAARAAAAEQAAAAAAAUBQAAAAAEAAAAEAAAAsP7//wgAAAAYAAAADAAAAHByb2Nlc3NfbmFtZQAAAAAEAAAAbmFtZQAAAAAAAAAApP7//wwAAABwcm9jZXNzX25hbWUAAAAAKv///xQAAABEAAAARAAAAAAABQFAAAAAAQAAAAQAAAAY////CAAAABgAAAANAAAAaW5zdGFuY2VfbmFtZQAAAAQAAABuYW1lAAAAAAAAAAAM////DQAAAGluc3RhbmNlX25hbWUAAACS////FAAAAEgAAABIAAAAAAAFAUQAAAABAAAABAAAAID///8IAAAAHAAAABEAAABtaWNyb3NlcnZpY2VfbmFtZQAAAAQAAABuYW1lAAAAAAAAAAB4////EQAAAG1pY3Jvc2VydmljZV9uYW1lABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAUAAAAFQAAAAAAAUBUAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAABwAAAARAAAAYXZhaWxhYmlsaXR5X3pvbmUAAAAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAARAAAAYXZhaWxhYmlsaXR5X3pvbmUAAAAAAAAA/////+gDAAAUAAAAAAAAAAwAFgAUABMADAAEAAwAAABICAAAAAAAABQAAAAAAAADAwAKABgADAAIAAQACgAAABQAAAC4AgAACgAAAAAAAAAAAAAAKgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAAAAAAAAAAMAAAAAAAAABwAAAAAAAAAKAAAAAAAAAAAAAAAAAAAACgAAAAAAAAADAAAAAAAAAA0AAAAAAAAABAAAAAAAAAABABAAAAAAAAAAAAAAAAAAAQAQAAAAAAADAAAAAAAAAAQAEAAAAAAABoAQAAAAAAAKgCAAAAAAAACAAAAAAAAACwAgAAAAAAADAAAAAAAAAA4AIAAAAAAAAAAAAAAAAAAOACAAAAAAAAAAAAAAAAAADgAgAAAAAAADAAAAAAAAAAEAMAAAAAAAAgAAAAAAAAADADAAAAAAAACAAAAAAAAAA4AwAAAAAAADAAAAAAAAAAaAMAAAAAAAAAAAAAAAAAAGgDAAAAAAAAAAAAAAAAAABoAwAAAAAAADAAAAAAAAAAmAMAAAAAAACgAAAAAAAAADgEAAAAAAAAAAAAAAAAAAA4BAAAAAAAADAAAAAAAAAAaAQAAAAAAABgAAAAAAAAAMgEAAAAAAAAAAAAAAAAAADIBAAAAAAAADAAAAAAAAAA+AQAAAAAAADoAAAAAAAAAOAFAAAAAAAAAAAAAAAAAADgBQAAAAAAADAAAAAAAAAAEAYAAAAAAABoAAAAAAAAAHgGAAAAAAAAAAAAAAAAAAB4BgAAAAAAAFAAAAAAAAAAyAYAAAAAAAAIAAAAAAAAANAGAAAAAAAAUAAAAAAAAAAgBwAAAAAAAAgAAAAAAAAAKAcAAAAAAAAwAAAAAAAAAFgHAAAAAAAAAAAAAAAAAABYBwAAAAAAAAAAAAAAAAAAWAcAAAAAAAAwAAAAAAAAAIgHAAAAAAAAcAAAAAAAAAD4BwAAAAAAAAAAAAAAAAAA+AcAAAAAAABQAAAAAAAAAAAAAAAPAAAACgAAAAAAAAAAAAAAAAAAAAoAAAAAAAAAAAAAAAAAAAAKAAAAAAAAAAAAAAAAAAAACgAAAAAAAAAKAAAAAAAAAAoAAAAAAAAAAAAAAAAAAAAKAAAAAAAAAAoAAAAAAAAACgAAAAAAAAAAAAAAAAAAAAoAAAAAAAAAAAAAAAAAAAAKAAAAAAAAAAAAAAAAAAAACgAAAAAAAAAAAAAAAAAAAAoAAAAAAAAAAAAAAAAAAAAKAAAAAAAAAAoAAAAAAAAACgAAAAAAAAAKAAAAAAAAAAoAAAAAAAAAAAAAAAAAAAAKAAAAAAAAAAAAAAAAAAAAAAAAAAsAAAAWAAAAIQAAACwAAAA3AAAAQgAAAE0AAABYAAAAYwAAAG4AAAAAAAAAdXMtZWFzdC0xLTF1cy1lYXN0LTEtMXVzLWVhc3QtMS0xdXMtZWFzdC0xLTF1cy1lYXN0LTEtMXVzLWVhc3QtMS0xdXMtZWFzdC0xLTF1cy1lYXN0LTEtMXVzLWVhc3QtMS0xdXMtZWFzdC0xLTEAAAAAAAAGAAAADAAAABIAAAAYAAAAHgAAACQAAAAqAAAAMAAAADYAAAA8AAAAAAAAAGFwb2xsb2Fwb2xsb2Fwb2xsb2Fwb2xsb2Fwb2xsb2Fwb2xsb2Fwb2xsb2Fwb2xsb2Fwb2xsb2Fwb2xsbwAAAAAAAAAAJAAAAEgAAABsAAAAkAAAALQAAADYAAAA/AAAACABAABEAQAAaAEAAAAAAABpLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb21pLXphWnN3bUprLWFwb2xsby0wMDAwLmFtYXpvbmF3cy5jb20AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAAAABgAAAAkAAAAMAAAADwAAABIAAAAVAAAAGAAAABsAAAAeAAAAAAAAAEFMMkFMMkFMMkFMMkFMMkFMMkFMMkFMMkFMMkFMMgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAACAAAAAwAAAAQAAAAFAAAABgAAAAcAAAAIAAAACQAAAAoAAAAAAAAAB1cy1lYXN0LTEtY2VsbC0xdXMtZWFzdC0xLWNlbGwtMXVzLWVhc3QtMS1jZWxsLTF1cy1lYXN0LTEtY2VsbC0xdXMtZWFzdC0xLWNlbGwtMXVzLWVhc3QtMS1jZWxsLTF1cy1lYXN0LTEtY2VsbC0xdXMtZWFzdC0xLWNlbGwtMXVzLWVhc3QtMS1jZWxsLTF1cy1lYXN0LTEtY2VsbC0xAAAAAAkAAAASAAAAGwAAACQAAAAtAAAANgAAAD8AAABIAAAAUQAAAFoAAAAAAAAAdXMtZWFzdC0xdXMtZWFzdC0xdXMtZWFzdC0xdXMtZWFzdC0xdXMtZWFzdC0xdXMtZWFzdC0xdXMtZWFzdC0xdXMtZWFzdC0xdXMtZWFzdC0xdXMtZWFzdC0xAAAAAAAAAAAAABcAAAAuAAAARQAAAFwAAABzAAAAigAAAKEAAAC4AAAAzwAAAOYAAAAAAAAAdXMtZWFzdC0xLWNlbGwtMS1zaWxvLTF1cy1lYXN0LTEtY2VsbC0xLXNpbG8tMXVzLWVhc3QtMS1jZWxsLTEtc2lsby0xdXMtZWFzdC0xLWNlbGwtMS1zaWxvLTF1cy1lYXN0LTEtY2VsbC0xLXNpbG8tMXVzLWVhc3QtMS1jZWxsLTEtc2lsby0xdXMtZWFzdC0xLWNlbGwtMS1zaWxvLTF1cy1lYXN0LTEtY2VsbC0xLXNpbG8tMXVzLWVhc3QtMS1jZWxsLTEtc2lsby0xdXMtZWFzdC0xLWNlbGwtMS1zaWxvLTEAAAAAAAAKAAAAFAAAAB4AAAAoAAAAMgAAADwAAABGAAAAUAAAAFoAAABkAAAAAAAAAHI1LjR4bGFyZ2VyNS40eGxhcmdlcjUuNHhsYXJnZXI1LjR4bGFyZ2VyNS40eGxhcmdlcjUuNHhsYXJnZXI1LjR4bGFyZ2VyNS40eGxhcmdlcjUuNHhsYXJnZXI1LjR4bGFyZ2UAAAAA5vZWC9BzV0AqQsElIj8UQMTF5jN9CzpAeWrYXWn3TECT+t/BGX5UQKmTNUCXp1RAo1u1RbIZMkB1A4cS/OIuQO4IH9RuCClAEj1DjQ5iWEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACwAAABYAAAAhAAAALAAAADcAAABCAAAATQAAAFgAAABjAAAAbgAAAAAAAABtZW1vcnlfdXNlZG1lbW9yeV91c2VkbWVtb3J5X3VzZWRtZW1vcnlfdXNlZG1lbW9yeV91c2VkbWVtb3J5X3VzZWRtZW1vcnlfdXNlZG1lbW9yeV91c2VkbWVtb3J5X3VzZWRtZW1vcnlfdXNlZAAAAJZS1Cb2FBYA5ArrSvYUFgAmTL9r9hQWADIoz4z2FBYA0jlWrvYUFgDeFWbP9hQWACBXOvD2FBYAED+eE/cUFgC+Svsz9xQWAMomC1X3FBYQAAAADAAUABIADAAIAAQADAAAABAAAAAsAAAAOAAAAAAAAwABAAAAUAcAAAAAAADwAwAAAAAAAEgIAAAAAAAAAAAAAAAAAAAAAAoADAAAAAgABAAKAAAACAAAAHQAAAADAAAATAAAACgAAAAEAAAAZPn//wgAAAAMAAAAAAAAAAAAAAAFAAAAcmVmSWQAAACE+f//CAAAAAwAAAAAAAAAAAAAAAQAAABuYW1lAAAAAKT5//8IAAAADAAAAAIAAAB7fQAABAAAAG1ldGEAAAAADwAAABgGAACUBQAAKAUAALwEAABYBAAA9AMAAJgDAAA8AwAA4AIAAHQCAADsAQAAYAEAAOQAAAB4AAAAGAAAAAAAEgAYABQAAAATAAwAAAAIAAQAEgAAABQAAAA8AAAAPAAAAAAAAAo8AAAAAQAAAAQAAAA4+v//CAAAABAAAAAEAAAAdGltZQAAAAAEAAAAbmFtZQAAAAAAAAAAKv7//wAAAwAEAAAAdGltZQAAAACm+v//FAAAAEQAAABEAAAAAAAFAUAAAAABAAAABAAAAJT6//8IAAAAGAAAAAwAAABtZWFzdXJlX25hbWUAAAAABAAAAG5hbWUAAAAAAAAAAIj6//8MAAAAbWVhc3VyZV9uYW1lAAAAAA77//8UAAAATAAAAEwAAAAAAAUBSAAAAAEAAAAEAAAA/Pr//wgAAAAgAAAAFgAAAG1lYXN1cmVfdmFsdWU6OnZhcmNoYXIAAAQAAABuYW1lAAAAAAAAAAD4+v//FgAAAG1lYXN1cmVfdmFsdWU6OnZhcmNoYXIAAIb7//8UAAAATAAAAFQAAAAAAAIBWAAAAAEAAAAEAAAAdPv//wgAAAAgAAAAFQAAAG1lYXN1cmVfdmFsdWU6OmJpZ2ludAAAAAQAAABuYW1lAAAAAAAAAAAIAAwACAAHAAgAAAAAAAABQAAAABUAAABtZWFzdXJlX3ZhbHVlOjpiaWdpbnQAAAAO/P//FAAAAEwAAABUAAAAAAADAVQAAAABAAAABAAAAPz7//8IAAAAIAAAABUAAABtZWFzdXJlX3ZhbHVlOjpkb3VibGUAAAAEAAAAbmFtZQAAAAAAAAAAAAAGAAgABgAGAAAAAAACABUAAABtZWFzdXJlX3ZhbHVlOjpkb3VibGUAAACS/P//FAAAAEQAAABEAAAAAAAFAUAAAAABAAAABAAAAID8//8IAAAAGAAAAA0AAABpbnN0YW5jZV90eXBlAAAABAAAAG5hbWUAAAAAAAAAAHT8//8NAAAAaW5zdGFuY2VfdHlwZQAAAPr8//8UAAAAPAAAADwAAAAAAAUBOAAAAAEAAAAEAAAA6Pz//wgAAAAQAAAABAAAAHNpbG8AAAAABAAAAG5hbWUAAAAAAAAAANT8//8EAAAAc2lsbwAAAABS/f//FAAAADwAAAA8AAAAAAAFATgAAAABAAAABAAAAED9//8IAAAAEAAAAAYAAAByZWdpb24AAAQAAABuYW1lAAAAAAAAAAAs/f//BgAAAHJlZ2lvbgAAqv3//xQAAAA8AAAAPAAAAAAABQE4AAAAAQAAAAQAAACY/f//CAAAABAAAAAEAAAAY2VsbAAAAAAEAAAAbmFtZQAAAAAAAAAAhP3//wQAAABjZWxsAAAAAAL+//8UAAAAQAAAAEAAAAAAAAUBPAAAAAEAAAAEAAAA8P3//wgAAAAUAAAACwAAAGpka192ZXJzaW9uAAQAAABuYW1lAAAAAAAAAADg/f//CwAAAGpka192ZXJzaW9uAGL+//8UAAAAQAAAAEAAAAAAAAUBPAAAAAEAAAAEAAAAUP7//wgAAAAUAAAACgAAAG9zX3ZlcnNpb24AAAQAAABuYW1lAAAAAAAAAABA/v//CgAAAG9zX3ZlcnNpb24AAML+//8UAAAARAAAAEQAAAAAAAUBQAAAAAEAAAAEAAAAsP7//wgAAAAYAAAADAAAAHByb2Nlc3NfbmFtZQAAAAAEAAAAbmFtZQAAAAAAAAAApP7//wwAAABwcm9jZXNzX25hbWUAAAAAKv///xQAAABEAAAARAAAAAAABQFAAAAAAQAAAAQAAAAY////CAAAABgAAAANAAAAaW5zdGFuY2VfbmFtZQAAAAQAAABuYW1lAAAAAAAAAAAM////DQAAAGluc3RhbmNlX25hbWUAAACS////FAAAAEgAAABIAAAAAAAFAUQAAAABAAAABAAAAID///8IAAAAHAAAABEAAABtaWNyb3NlcnZpY2VfbmFtZQAAAAQAAABuYW1lAAAAAAAAAAB4////EQAAAG1pY3Jvc2VydmljZV9uYW1lABIAGAAUABMAEgAMAAAACAAEABIAAAAUAAAAUAAAAFQAAAAAAAUBUAAAAAEAAAAMAAAACAAMAAgABAAIAAAACAAAABwAAAARAAAAYXZhaWxhYmlsaXR5X3pvbmUAAAAEAAAAbmFtZQAAAAAAAAAABAAEAAQAAAARAAAAYXZhaWxhYmlsaXR5X3pvbmUAAABoBwAAQVJST1cx
